CHANGELOG.md

```diff
@@ -12,13 +12,17 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ### Changed
 
-- Timestamp argument accepts input in string type( format `yyyy-MM-dd HH:MM:SS` ), int or float type( epoch time ) as well as a datetime instance for following:
+- Timestamp arguments accept input in string type( format `yyyy-MM-dd HH:MM:SS` ), int or float type( epoch time, e.g. using `time.time()` ) as well as a `datetime` instance for the following:
```

Comment:
There are some other errors with this that I address and that got merged.

For instance:

1. Don't need to mention float again as they are already supported
2. The only new supported type for `departure_date` is `datetime`. `str` was already supported.

Reply:
If you look at what is in master now..

Reply:
thanks, I'll adjust accordingly